data/src/main/java/me/fontys/semester4/data/repository/PostalcodePartRepository.java

```diff
@@ -4,6 +4,10 @@
 import org.springframework.data.jpa.repository.JpaRepository;
 import org.springframework.data.jpa.repository.JpaSpecificationExecutor;
 
+import java.util.Optional;
+
 public interface PostalcodePartRepository extends JpaRepository<PostalcodePart, Long>, JpaSpecificationExecutor<PostalcodePart> {
+    Optional<PostalcodePart> findByPostalcodeAndEvenAndStarthousenrAndEndhousenr(String Postalcode
+    ,String Even, Long Starthousenr, Long Endhousenr);
```

Comment:
if lines get too long for any linter to make a comment about it (usually a length of 80 or 120), please format like so:
```java
Optional<PostalcodePart> findByPostalcodeAndEvenAndStarthousenrAndEndhousenr(
    String Postalcode,
    String Even, 
    Long Starthousenr, 
    Long Endhousenr
);
```
to keep it readable :)

Reply:
I will change it.